roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -0,0 +1,34 @@
+# These tasks are run when contiv_network_mode is set to aci
+
+- name: check aci-gw container image
+  shell: docker inspect contiv/aci-gw
+  register: docker_aci_inspect_result
+  ignore_errors: yes
+  run_once: true
+
+- name: pull aci-gw container
+  shell: docker pull contiv/aci-gw
+  when: "'No such image' in docker_aci_inspect_result.stderr"
+  run_once: true
+  tags:
```

Comment:
I don't think we should prebake aci-gw container because this image will need to be released along side with netmaster/netplugin. So we need to pull the corresponding version every time contiv_network is deployed.

On related note, we should be pulling a specific version of aci-gw container here.